kaspresso/src/main/kotlin/com/kaspersky/kaspresso/device/bluetooth/BluetoothImpl.kt

```diff
@@ -0,0 +1,137 @@
+package com.kaspersky.kaspresso.device.bluetooth
+
+import android.bluetooth.BluetoothAdapter
+import android.bluetooth.BluetoothManager
+import android.content.Context
+import com.kaspersky.components.kautomator.system.UiSystem
+import com.kaspersky.kaspresso.device.server.AdbServer
+import com.kaspersky.kaspresso.flakysafety.algorithm.FlakySafetyAlgorithm
+import com.kaspersky.kaspresso.internal.exceptions.AdbServerException
+import com.kaspersky.kaspresso.internal.systemscreen.NotificationsFullScreen
+
+import com.kaspersky.kaspresso.logger.UiTestLogger
+import com.kaspersky.kaspresso.params.FlakySafetyParams
+
+/**
+ * The implementation of the [Bluetooth] interface.
+ */
+class BluetoothImpl(
```

Comment:
if this class is only needed inside our framework and should not be visible outside - let's make it internal to limit the creation of new instances and minimize the side effects of possible breaking changes in this class